tests/mitoai_ui_tests/mitoai.spec.ts

```diff
@@ -72,6 +72,18 @@ test.describe('Mito AI Chat', () => {
     await expect(page.locator('.cm-codeDiffInsertedStripe')).toBeVisible();
   });
 
+  test('Code diffs are automatically rejected before new messages are sent', async ({ page }) => {
```

Comment:
Let's add a test to make sure we don't overwrite users's code (ie: the bug above)